### ISSUE TITLE
Unresolved conflict Merged file not notified

### ISSUE THREAD

**Issue by estebanfeldman** [ForkIssues/TrackerWin#1618]
I work with Unity for Game Development.

I had this persistent issue after merging master into one of my branches that Unity was giving me an strange error and I couldn't pinpoint what was the problem since the file in question has never been modified by me.

I had to go to the actual file and open it in a text editor.

I found that there are unresolved conflicts in a file that I was not notified about and was merged with the conflicts. 
I was doubtful, maybe it was one of the artists that don't know how to use git but on checking the file history there is the original commit and the merge into my branch.

![image](https://user-images.githubusercontent.com/157223/184082638-0d93a4a0-30f5-49c4-9c48-e8102ebb6080.png)

Is there a way to avoid this in the future?

Thanks

**Comment by DanPristupov** [comment]
> Is there a way to avoid this in the future?

Yes. Try to reproduce the problem in an empty repo and send it to me (support@fork.dev) as a zip file.

**Comment by DanPristupov** [comment]
I think some other tool might be involved here.
